test/saithrift/tests/sail3.py

```diff
@@ -48,6 +48,9 @@ def runTest(self):
         nhop1 = sai_thrift_create_nhop(self.client, addr_family, ip_addr1, rif_id1)
         sai_thrift_create_route(self.client, vr_id, addr_family, ip_addr1_subnet, ip_mask1, rif_id1)
 
+        # Waiting the end of configuration ...
+        wait_till_configuration_will_end()
```

Comment:
We added a delay to wait until configuration is fully populated to SAI adapter on the server side.
Otherwise test cases fail on our Silicon Emulator. We believe this happens due to a fact that we run both test client and server on one PC and a transport delay is minimal.